ggshield/cmd/secret/scan/prereceive.py

```diff
@@ -66,10 +64,10 @@ def _execute_prereceive(
         )
         if return_code:
             click.echo(
-                remediation_message(
-                    remediation_steps=config.user_config.secret.prereceive_remediation_message,
-                    bypass_message=BYPASS_MESSAGE,
-                    rewrite_git_history=True,
+                (
+                    config.user_config.secret.prereceive_remediation_message
```

Comment:
equivalent to: `config.user_config.secret.prereceive_remediation_message or client.remediation_messages.pre_receive`